dev/com.ibm.ws.org.apache.taglibs.standard/bnd.bnd

```diff
@@ -28,7 +28,7 @@ Export-Package: \
 
 Include-Resource: \
    @${repo;com.ibm.ws.javax.j2ee:jsp.tld.standard;1.2}!/META-INF/*.tld,\
-   @${repo;com.ibm.ws.org.apache:xalan;2.7.2}!/!(META-INF/*|org/apache/xalan/xsltc/*|java_cup/*|trax/*|org/apache/bcel/*|org/apache/regexp/*)
+   @${repo;com.ibm.ws.org.apache:xalan;2.7.3}!/!(META-INF/*|org/apache/xalan/xsltc/*|java_cup/*|trax/*|org/apache/bcel/*|org/apache/regexp/*)
```

Comment:
Updating the copyright header here to include 2024 would also be great!

Reply:
In addition do we need to exclude these packages any longer? @isaacrivriv

Reply:
The commented out test is fixed in this [PR #28939](https://github.com/OpenLiberty/open-liberty/pull/28939).
Also, updated the Copyright header to include 2024.

Reply:
The failing test is fixed through this PR [#28939](https://github.com/OpenLiberty/open-liberty/pull/28939).
Also, updated the Copyright header to include 2024.